packages/sanity/src/core/releases/plugin/index.ts

```diff
@@ -26,12 +26,13 @@ export const releases = definePlugin({
       title: 'Releases',
       component: ReleasesTool,
       router: route.create('/', [route.create('/:bundleId')]),
-      canHandleIntent: (intent, params) => {
-        return Boolean(intent === 'release' && params.slug)
+      canHandleIntent: (intent) => {
+        // If intent is release, open the releases tool.
+        return Boolean(intent === 'release')
```

Comment:
Instead of not resolving the intent if the params are missing, resolve to the tool at root level.